Create v24.2.0_cycle25.mdx (#263)
seqeralabs/docs: changelog/seqera-cloud/v24.2.0_cycle25.mdx

---
title: Seqera Cloud v24.2.0_cycle25
date: 2024-10-14
tags: [seqera cloud]
---

### Feature updates and improvements

> - Quick launch is now error resilient when adding a pipeline that does not have a master branch.

Reviewer: ```suggestion
- Quick launch now accepts pipelines that do not have a branch named 'master'.
```
Not having a `master` branch is not necessarily an error that we need to beware of. Maybe we can re-phrase this.

Reviewer 2: @JaimeSeqLabs, what's an inverse way of stating this? It accepts pipelines with any branch name? If previously users knew that pipelines without a `master` branch were a incompatible with quick launch, seeing that a pipeline without this now works is explicit in revealing that that specific issue is resolved. What do we actually what to say here? Thanks!

Reviewer: I just wanted to make more clear that a pipeline without `master` branch does not cause an error from which Platform is recovering (error resilient).

- Quick launch is now error resilient when adding a pipeline that does not have a master branch.
- Improved query times on some clean up job queries.
- NF Launcher image is now by default pulled from `cr.seqera.io` public registry instead of quay.
- Data studio sessions can now share mounted data across the same workspace.
- The new Launch Form is now available in Quick Launch and personal workspaces.

> - Improved fetching of the cloud cost estimation for GCP's spot instances.

Reviewer 2: ```suggestion
- Improved fetching of the cloud cost estimation for Google Cloud Platform's (GCP) spot instances.
```
